openwillis/measures/text/speech_attribute.py

```diff
@@ -374,8 +355,11 @@ def speech_characteristics(json_conf, language="en", speaker_label=None, min_tur
 
         if bool(json_conf):
             language = language[:2].lower() if (language and len(language) >= 2) else "na"
+            if language not in measures["english_langs"] + measures["supported_langs_sentence_embeddings"] + measures["supported_langs_bert"]:
+                logger.warning(f"Language {language} not supported. Defaulting to English")
+                language = "en"
```

Comment:
Just to confirm: We will compute all the language-dependent variables if a user passes an unsupported language code, regardless of the original language

Reply:
that's what I thought we could do yes. If they input a supported language that is not english, we compute only the non-english needed measures, if they input an unsupported language code we act as if the language is English. But maybe it makes more sense to throw an error and stop the calculation. I'm not sure what do you think?